lib/active_merchant/billing/gateways/priority.rb

```diff
@@ -182,6 +183,17 @@ def add_type_merchant_purchase(params, merchant, is_settle_funds, options)
         params['sourceZip'] = options[:billing_address][:zip] if options[:billing_address]
         params['taxExempt'] = false
         params['tenderType'] = 'Card'
+
+        params['cardholderPresence'] = ''
+        params['deviceAttendance'] = ''
+        params['deviceInputCapability'] = ''
+        params['deviceLocation'] = ''
+        params['panCaptureMethod'] = ''
+        params['partialApprovalSupport'] = ''
+        params['pinCaptureCapability'] = ''
+        params['shipAmount'] = options[:shipAmount]
+        params['shipToCountry'] = options[:shipToCountry]
+        params['shipToZip'] = options[:shipToZip]
```

Comment:
Please change the keys in the `options` hash to snake_case, e.g. `params['shipToZip'] = options[:ship_to_zip]`

Reply:
Changed to snake-case.